ci_support/run_docker_build.sh

```diff
@@ -70,4 +70,11 @@ yum install -y devtoolset-2-gcc-gfortran
     set +x
     conda build /recipe_root --quiet || exit 1
     upload_or_check_non_existence /recipe_root conda-forge --channel=main || exit 1
+
+    set -x
+    export CONDA_NPY=110
```

Comment:
I don't think we plan to maintain NumPy 1.10 for much longer. Though I don't believe it has been formally discussed, I expect we will drop it after NumPy 1.12 lands.

As for Python 3.6, in particular, we do run the test suite in both builds of NumPy and SciPy. So if it passed, it should work, right? Were there many notable changes needed to NumPy and SciPy for Python 3.6 support?